packages/purpose-process/src/utilities/errorMappers.ts

```diff
@@ -152,3 +152,16 @@ export const createReversePurposeErrorMapper = (
     .with("duplicatedPurposeTitle", () => HTTP_STATUS_CONFLICT)
     .with("tenantNotFound", () => HTTP_STATUS_INTERNAL_SERVER_ERROR)
     .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
+
+export const clonePurposeErrorMapper = (error: ApiError<ErrorCodes>): number =>
+  match(error.code)
+    .with("purposeNotFound", () => HTTP_STATUS_NOT_FOUND)
+    .with("tenantKindNotFound", () => HTTP_STATUS_FORBIDDEN)
```

Comment:
I think that this should be a 400, since forbidden is usually used when the caller don't have sufficient rights

Reply:
Updated